packaging/ansible-runner-service-project/project/roles/ovirt-host-deploy-misc/tasks/tuned_profile.yml

```diff
@@ -1,28 +1,35 @@
 ---
 - name: Set profile
   set_fact:
-    profile: >
-      "{{ 'virtual-host' if (host_deploy_tuned_profile is none and host_deploy_virt_enabled|bool)
-      else host_deploy_tuned_profile }}"
+    profile:
+      "{{ 'virtual-host'
```

Comment:
wouldn't it be easier to just do this in the java code? 
just change one line https://github.com/oVirt/ovirt-engine/blob/4a9c155542eae7efec677d8384b3243ec31514a5/backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/hostdeploy/InstallVdsInternalCommand.java#L280

Reply:
the problem is not in this ansible code, it's the "" parameter that we pass to it. With a wrong parameters it's ok that this role fails.